### DISCUSSION TITLE
Need help uninstalling

### DISCUSSION THREAD

**Discussion by emsss78** [flybywiresim/aircraft#6172] [Help]
So all of a sudden my FBW A320 Experimental version started acting crazy....after take off the speed of the plane drops like crazy, autopilot disconnects and I can't re-engage it. I had around 200 hours of flights with the FBW and never had a problem before. I'm even thinking to re-install the sim....this is so weird everything was working perfectly until some fbw installer update messed it up.

Can you guide me to what folders I need to delete other than the one in the community folder??

**Comment by CptBayer** [comment]
Hi ...

.....\AppData\Local\Packages\Microsoft.FlightSimulator_8wekyb3d8bbwe\LocalState\packages

Look also here ... I had also a crazy behaviour and a new install via the FBW installer solved it ... save maybe the throttleconfig.ini .... 

greets

**Comment by emsss78** [comment]
Alrighty...I did keep a backup of Throttleconfig.ini...Anyways i'll do a re-install using the installer and see how it goes, not sure why it got messed up.
